scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -155,13 +194,17 @@ def get_logical_node_class(self):
     check_type(logical_node_class, list, "logical_node_class")
     return logical_node_class
 
-def get_storage(self):
+def get_local_storage(self):
```

Comment:
add a comment, "getting storage for the current node" or something relevant

Reply:
Done